utils.py

```diff
@@ -841,6 +841,36 @@ def compute_list_difference(list_a, list_b):
     return list(set(list_a) - set(list_b))
 
 
+def is_appengine_development_environment():
```

Comment:
Isn't this what ``DEV_MODE`` is meant for? Why can't we use that for distinguishing between dev & prod?

Reply:
@kevjumba I suggest adding an explicit comment in the docstring to explain this clearly.